qingstor/qsctl/constants.py

```diff
@@ -29,3 +31,9 @@
 
 # tqdm bar format
 BAR_FORMAT = "{l_bar}{bar}| {n_fmt}/{total_fmt} [{remaining} ETA, {rate_fmt}]"
+
+# In python2.6, tqdm will raise encoding error, so use ascii instead
+USE_ASCII = is_python26
+
+# Units used in output
+UNITS = ('KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB')
```

Comment:
`'KB', 'MB', 'GB', 'TB', 'PB', 'EB'` use 1000 as carry, and `'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB'` use 1024, which is more correct.

Ref: https://en.wikipedia.org/wiki/Mebibyte